karma.conf.js

```diff
@@ -25,6 +25,8 @@ webpackConfig.module.rules = webpackConfig.module.rules.filter((rule) => {
   return !rule.use || !rule.use.some(use => use.loader === '@folio/stripes-config');
 });
 
+webpackConfig.plugins = [];
```

Comment:
https://github.com/folio-org/stripes-core/pull/120 caused our test run to load in multiple `webpack.EnvironmentPlugin`s. The result was a full run of the tests went from about ~1min to ~5min. Clearing out the plugins and starting from scratch resolved the slowdown.